lib/cucumber/events/step_activated.rb

```diff
@@ -20,6 +20,8 @@ class StepActivated < Core::Event.new(:test_step, :step_match)
       def initialize(test_step, step_match)
         @test_step = test_step
         @step_match = step_match
+
+        super
```

Comment:
Whilst this is a rubocop requirement, it is also an unsafe auto-correct. So I just wanted to check we have coverage for this?

Reply:
I took a look at the source code of the parent class: https://github.com/cucumber/cucumber-ruby-core/blob/main/lib/cucumber/core/event.rb

It seems to be safe here.

Don't know if we have coverage for this. But as this is a center piece of cucumber-ruby, I thing we may have notice issues.